src/main/java/pl/piekoszek/gorskimatches/challange/ChallengeScoreAndAnswers.java

```diff
@@ -12,6 +12,11 @@ class ChallengeScoreAndAnswers {
 
     private List<Integer> scoreUser2;
 
+    public float timeUser1;
+
+    public float timeUser2;
```

Comment:
also it's not completely obvious what exactly `time` means, you could try more descriptive name